drupal.make.yml

```diff
@@ -25,12 +25,16 @@ projects:
     version: ~
   features_extra:
     version: ~
+  features_override:
```

Comment:
What is the correct module to use to determine the code differences in features?  I tried:

features_override
diff